source/main/utils/setupLogging.js

```diff
@@ -9,12 +9,12 @@ export const setupLogging = () => {
   const logFilePath = path.join(pubLogsFolderPath, APP_NAME + '.log');
   ensureDirectoryExists(pubLogsFolderPath);
 
-  Log.transports.console.level = 'warn';
-  Log.transports.rendererConsole.level = 'warn';
-  Log.transports.file.level = 'debug';
-  Log.transports.file.maxSize = 20 * 1024 * 1024;
-  Log.transports.file.file = logFilePath;
-  Log.transports.file.format = '[{y}-{m}-{d} {h}:{i}:{s}.{ms} {z}] [{level}] {text}';
+  log.transports.console.level = false;
```

Comment:
@darko-mijic @DominikGuzei `console` transport has to be disabled as it causes `macOS` installer issues (Alert dialog is shown on first `log.error` call)